spec.md

```diff
@@ -1422,8 +1431,21 @@ message Error {
       // terminated or deleted before retrying.
       NODE_DOES_NOT_EXIST = 6;
 
+      // Indicates that the specified `NodeID` is not allowed or
+      // understood by the Plugin. More human-readable information MAY
+      // be provided in the `error_description` field.
+      //
+      // Recovery behavior: Caller MUST fix the `NodeID` before
+      // retrying.
       INVALID_NODE_ID = 7;
+
       VOLUME_NOT_ATTACHED_TO_SPECIFIED_NODE = 8;
+
+      // Indicates that the Plugin does not support the operation
+      // without a `NodeID`.
+      //
+      // Recovery behavior: Caller MUST fix the `NodeID` before
```

Comment:
Done. I used the same wording as in `ControllerPublishVolumeError::VOLUME_ALREADY_PUBLISHED`.